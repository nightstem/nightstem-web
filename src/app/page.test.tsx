import { axe } from 'vitest-axe';
import { expect, it } from 'vitest';
import { render } from '@testing-library/react';

import Page from './page';

it('matches snapshot', () => {
  const { container } = render(<Page />);
  expect(container).toMatchSnapshot();
});

it('does not have violations', async () => {
  const { container } = render(<Page />);
  const result = await axe(container);
  expect(result).toHaveNoViolations();
});
