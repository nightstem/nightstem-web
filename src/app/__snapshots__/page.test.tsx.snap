// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`matches snapshot 1`] = `
<div>
  <div
    class="w-full h-full"
  >
    Homepage
  </div>
</div>
`;
